### ISSUE TITLE
[MAS3.2.1][Keyboard Navigation-File->New bot configuration] When the user press the enter key on “New bot configuration “ focus is not on the Bot name text field.

### ISSUE THREAD

**Issue by Kumar2608** [microsoft/BotFramework-Emulator#1805]
**Actual Result:**
When the user press the enter key on “New bot configuration “ focus is not on the Bot name text field 

**Expected Result:**
When the user press the enter key on “New bot configuration “ focus should be on the Bot name text field.

**Repro Steps:**
1.Open the Bot Framework V4 Emulator.
2.Navigate to the File menu and select it
3.File menu gets opened  
4.Navigate to the New bot configuration and select it 
5.New bot configuration dialog gets opened 
6.Observe the issue

**User Impact:**
Users who rely on keyboard navigation will be affected to navigate the control if keyboard focus is not lands on the expected control keyboard users will be affected.

**Test Environment:**
OS Build: Version 1903(OS Build 18362.239)
Application: Bot Framework V4 Emulator

**MAS Reference Link:**
https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={a4bbd37d-ee11-485e-b5c8-0539f0ac94b2}
[MAS3.2.1_File.zip](https://github.com/microsoft/BotFramework-Emulator/files/3562173/MAS3.2.1_File.zip)
